lib/app/features/home/home_widgets/home_screen_date_picker/home_screen_date_picker.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:date_picker_timeline/date_picker_widget.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:score_live/app/features/home/cubit/home_cubit.dart';
+import 'package:score_live/app/features/home/home_tabs/score_tab/cubit/score_tab_cubit.dart';
+import 'package:score_live/core/applocalization_context.dart';
+import 'package:score_live/presentation/constants/app_colors.dart';
+
+class HomeScreenDatePicker extends StatefulWidget {
```

Comment:
Do you need this to be `StatefulWidget` ?

Reply:
fixed